src/main/java/ch/njol/skript/test/runner/TestTracker.java

```diff
@@ -54,6 +55,12 @@ public static void testFailed(String msg) {
 		failedTests.put(currentTest, msg);
 	}
 
+	public static void testFailed(String msg, Script script) {
+		String file = script.getConfig().getFileName();
+		file = file.substring(file.lastIndexOf("\\") + 1);
```

Comment:
Does this work on Linux machine? shouldn't it use `File.separator`

Reply:
No, see how using File.separator causes issues from this existing pull request fix https://github.com/SkriptLang/Skript/pull/5300

Reply:
What's the status of this convo? (Pickle's question)